src/Data/List/Membership/Setoid/Properties.agda

```diff
@@ -50,6 +52,31 @@ module _ {c ℓ} (S : Setoid c ℓ) where
   ∉-resp-≋ : ∀ {x} → (x ∉_) Respects _≋_
   ∉-resp-≋ xs≋ys v∉xs v∈ys = v∉xs (∈-resp-≋ (≋-sym xs≋ys) v∈ys)
 
+------------------------------------------------------------------------
+-- Irrelevance
+
+module _ {c ℓ} (S : Setoid c ℓ) where
+
+  open Setoid S
+  open Unique S
+  open Membership S
+
+  private
+    _≉_ = λ x y → ¬ (x ≈ y)
+
```

Comment:
@MatthewDaggitt Would it make sense to add this definition to `Setoid`?

Reply:
Yup, I included it in `IndexedSetoid`.